include/zmq.h

```diff
@@ -371,6 +374,18 @@ ZMQ_EXPORT const char *zmq_msg_gets (const zmq_msg_t *msg, const char *property)
 #define ZMQ_VMCI_BUFFER_MAX_SIZE 87
 #define ZMQ_VMCI_CONNECT_TIMEOUT 88
 #define ZMQ_USE_FD 89
+#define ZMQ_PEER_EVENTS 90
+
+/*  getsockopt additional struct to recover peer events                       */
+typedef struct zmq_gso_peer_events_t
+{
+    void *zid;
+    int zid_len;
+    short events;
+    short padding;
+}  zmq_gso_peer_events_t;
+
+#define ZMQ_GSO_PEER_EVS_SIZE (size_t)sizeof(zmq_gso_peer_events_t)
```

Comment:
What does EVS mean?

The type of sizeof(T) is defined to be size_t (http://en.cppreference.com/w/cpp/language/sizeof), so the cast is redundant.

Reply:
EVS = events. Again trying to shorten names.

Agree on the redundancy comment. Saw it afterwards

Reply:
Apart from the general abbreviations discussion, it would be a really good idea to use the same spelling as the name of the type this refers to.